core/templates/dev/head/pages/exploration_editor/feedback_tab/thread_table_directive.html

```diff
@@ -22,7 +22,7 @@
     </td>
     <td class="col-lg-3 col-md-3 col-sm-3">
       <span ng-if="thread.original_author_username"><[thread.original_author_username]></span>
-      <span ng-if="thread.suggestion_id"><[thread.author_name]></span>
+      <span ng-if="thread.originalAuthorName"><[thread.originalAuthorName]></span>
```

Comment:
These seem a tad weird to me -- why do we have both original_author_username and originalAuthorName?